crates/swc_css_minifier/src/compress/empty.rs

```diff
@@ -0,0 +1,86 @@
+use swc_css_ast::*;
+use swc_css_visit::{VisitMut, VisitMutWith};
+
+pub fn compress_empty() -> impl VisitMut {
+    CompressEmpty {}
+}
+
+struct CompressEmpty {}
+
+impl VisitMut for CompressEmpty {
+    fn visit_mut_stylesheet(&mut self, stylesheet: &mut Stylesheet) {
+        stylesheet.visit_mut_children_with(self);
+
+        stylesheet.rules.retain(|rule| match rule {
+            Rule::QualifiedRule(QualifiedRule { block, .. })
+            | Rule::AtRule(AtRule::FontFace(FontFaceRule { block, .. }))
+            | Rule::AtRule(AtRule::Keyframes(KeyframesRule { block, .. }))
+            | Rule::AtRule(AtRule::Media(MediaRule { block, .. }))
+            | Rule::AtRule(AtRule::Supports(SupportsRule { block, .. }))
+            | Rule::AtRule(AtRule::Page(PageRule { block, .. }))
+            | Rule::AtRule(AtRule::Viewport(ViewportRule { block, .. }))
+            | Rule::AtRule(AtRule::Document(DocumentRule { block, .. }))
+            | Rule::AtRule(AtRule::Property(PropertyRule { block, .. }))
+                if block.value.is_empty() =>
+            {
+                return false;
+            }
+            Rule::AtRule(AtRule::Layer(LayerRule { block, .. })) if block.is_some() => {
```

Comment:
I think it can be `Rule::AtRule(AtRule::Layer(LayerRule { block: Some(block), .. }))` instead.